rancher/dashboard
# Embed project monitoring metric dashboards

### Summary
This embeds the project monitoring Grafana metric dashboards if they are present and user has no access to cluster monitoring.

Fixes #7286

### Technical notes summary
To test:
* Create a cluster
* Install cluster monitoring
* Install project federator
* Install project monitoring into the default project
* Create a user in Rancher that only has access to the default project in the cluster as a project member
* Deploy a workload into the default project
* Check that the limited user can see grafana dashboards under the pod detail page and workload detail page metrics tabs

### Areas or cases that should be tested
Pod detail page and workload detail page.

### Areas which could experience regressions
Embedding of Grafana metrics dashboards on pod and workload detail pages

### Screenshot/Video
<img width="1972" alt="Bildschirmfoto 2023-06-09 um 16 25 20" src="https://github.com/rancher/dashboard/assets/243056/468bf665-3781-46b1-94cf-23be766891c1">
